resources/deu.ini

```diff
@@ -282,10 +289,10 @@ clone.formworld=Grundstückswelt
 
 ; the /p kick Subcommand
 kick.name=kick
-kick.desc=Kicke einen Spieler von deinem Plot
+kick.desc=Kicke einen Spieler von deinem Grundstück
 kick.usage=/p kick <spieler: target>
 kick.noPlayer=Der angegebene Spieler konnte nicht gefunden werden
-kick.notInPlot=Der angegebene Spieler befindet sich nicht auf deinem Plot
+kick.notInPlot=Der angegebene Spieler befindet sich nicht auf deinem Grundstück
```

Comment:
This change is outside the scope of this PR

Reply:
Oops, you're right.